examples/ipv6only-app/esp32/README.md

```diff
@@ -0,0 +1,109 @@
+# CHIP ESP32 IPV6 Only Example Application
+
+This application implements ESP32 wifi control to support IPV6 tests.
+
+Once connected the application acts as a UDP echo server and will echo udp
+messages it receives, this can be used to test for diconnect events.
+
+-   [CHIP ESP32 IPV6 Only Example Application](#chip-esp32-ipv6-only-example-application)
+    -   [Building the Example Application](#building-the-example-application)
+        -   [To build the application, follow these steps:](#to-build-the-application-follow-these-steps)
+    -   [Testing the Example Application](#testing-the-example-application)
+
+---
+
+## Building the Example Application
+
+Building the example application requires the use of the Espressif ESP32 IoT
```

Comment:
Can we link to common instructions here?

Reply:
This is a good point, but is the same problem for all platforms and examples, I'll put some thought into this for a follow up CL.